src/library_webgl.js

```diff
@@ -1864,11 +1867,13 @@ for (/**@suppress{duplicate}*/var i = 0; i <= {{{ GL_POOL_TEMP_BUFFERS_SIZE }}};
       return;
     }
 #endif
+#if INCLUDE_WEBGL1_FALLBACK
```

Comment:
It looks like this is reached even without the condition on `settings.MIN_WEBGL_VERSION`, unlike all those before it, so this if is not actually for the complement condition, IIUC?

Reply:
WEBGL_USE_GARBAGE_FREE_APIS itself is conditional on MIN_WEBGL_VERSION.

The idea is that INCLUDE_WEBGL1_FALLBACK will be set only in cases where `WEBGL_USE_GARBAGE_FREE_APIS` is set and also we know that we have only webgl2 support.  In this case that means that above return on line 1867 is guaranteed to run.

Reply:
The exact same pattern repeats below on line 1886

Reply:
I see, thanks, I was confused by the condition in link.py. I wrote a suggestion to simplify it below.